### DISCUSSION TITLE
More utility functions needed.

### DISCUSSION THREAD

**Discussion by unravel-dev** [xissburg/edyn#122] [General]
Hi, i think more utility functions should be available by the library out of the box.
For example i would like to be able to update for example the gravity of a rigidbody after it was created, but i cannot find such function.
Or i would like to  "unmake" a rigidbody, but i have to check inside the functions to see what components were added by make_rigidbody to make sure i remove all of them.
Or maybe change the material, but not knowing which components are affected internally by that.

For a simple example if i want to remove the gravity of an entity, i would probably need to zero the velocity as well.

Basically all of the properties defined by rigidbody_def should have a way to change at runtime.

**Comment by xissburg** [comment]
Thanks for the suggestion. It certainly needs more utility functions such as these. I will look into how the ones you mentioned could be properly integrated eventually.
